Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -98,6 +99,9 @@ namespace Terrain
                     ;
 
                 editContext->Class<ClipmapConfiguration>("Clipmap", "Settings related to clipmap rendering")
+                    ->DataElement(AZ::Edit::UIHandlers::CheckBox, &ClipmapConfiguration::m_clipmapEnabled,
+                        "Clipmap Enabled",
+                        "Enable the clipmaps for rendering terrain materials")
```

Comment:
Suggestion: Consider using this instead, so that the enabled toggle appears on the same line as the group expander:
```
                    ->GroupElementToggle("Clipmap Enabled", &ClipmapConfiguration::m_clipmapEnabled)
                    ->Attribute(AZ::Edit::Attributes::AutoExpand, false)
                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, AZ::Edit::PropertyRefreshLevels::AttributesAndValues)
```
You might also need to add an AutoExpand here:
```
                    ->DataElement(AZ::Edit::UIHandlers::Default, &TerrainWorldRendererConfig::m_clipmapConfig, "Clipmap configuration", "")
                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
```

Reply:
Thanks for the advice. I'll make a change.